<html>
<!-- ####################################################### -->
<!-- #  This html was produced by the igemwiki generator   # -->
<!-- #  https://github.com/igemuoftATG/generator-igemwiki  # -->
<!-- ####################################################### -->

<!-- repo for this wiki: https://github.com/igemuoftATG/wiki17 -->

</html>
{{Toronto/head}}
<html>
</html>
{{Toronto/navbar-dark-cyan}}
<html>

<!-- Header div -->
<div class="section">
	<div class="container header" id="dark-red">
		<h1>Notebook</h1>
	</div>
</div>

<!-- Content div -->
<div class="section bg-white">
	<div class="container content-page row">
		<div class="block content">

			<div id='prep' class='subsection'><h2 class="text-red"><a id="Preparing_stocks_plates_cell_cultures_etc_0"></a>Preparing stocks, plates, cell cultures, etc.</h2>
<h3><a id="Week_1_day_1_2606_2"></a>Week 1, day 1 (26/06)</h3>
<ul class="bullets">
<li>Prepared wash bottles of 70% Ethanol</li>
<li>Made 4x500mL LB media
<ul class="bullets">
<li>2 with agar for plating</li>
<li>2 without agar for preparing overnight cultures</li>
</ul>
</li>
<li>Made chloramphenicol (CAM) stocks</li>
<li>Plated out LB + agar onto sterile plates with CAM</li>
</ul>
<h3><a id="Week_1_day_2_2706_10"></a>Week 1, day 2 (27/06)</h3>
<ul class="bullets">
<li>Made 10mL 0.2M acetic acid/1M NaOH</li>
<li>Prepared 50mL of RF1/RF2 buffer (to make RbCl competent cells)</li>
<li>Streaked 4 plates (stored in 4°C):
<ul class="bullets">
<li>MG1655ΔLacI onto 2 LB and 1 LB+CAM plates</li>
<li>DH10β onto 1 LB plate</li>
</ul>
</li>
</ul>
<h3><a id="Week_1_day_3_2806_17"></a>Week 1, day 3 (28/06)</h3>
<p>Streaked 2 LB plates: MG1655ΔLacI &amp; DH10β to replace Tuesday’s plates (because they over-grew)</p>
<ul class="bullets">
<li>New plates put into 37°C incubator overnight, backup plates placed into 4°C fridge</li>
</ul>
<p>Prepared 5mL LB cultures and placed in overnight shaker of:</p>
<ul class="bullets">
<li>MG1655ΔLacI</li>
<li>DH10β</li>
</ul>
<h3><a id="Week_1_day_4_2906_25"></a>Week 1, day 4 (29/06)</h3>
<ul class="bullets">
<li>Prepared SOC medium</li>
<li>Cultured DH10β cells in baffled flasks until OD reached 0.4-0.55</li>
</ul>
<h3><a id="Week_1_day_5_3006_29"></a>Week 1, day 5 (30/06)</h3>
<ul class="bullets">
<li>Plated DH10β cells and incubated at 37°C overnight</li>
</ul>
<h3><a id="Week_1_day_6_3106_32"></a>Week 1, day 6 (31/06)</h3>
<ul class="bullets">
<li>Checked on plated cells from 30/06:</li>
</ul>
<h3><a id="Week_3_day_1_1007_47"></a>Week 3, day 1 (10/07)</h3>
<ul class="bullets">
<li>Made and autoclaved LB and LB+CAM liquid.</li>
<li>Poured 13 LB plates.
<ul class="bullets">
<li>LB+CAM plates were unsuccessful since incorrect amount of CAM was added (12.5g instead of 12.5mg).</li>
</ul>
</li>
</ul>
<h3><a id="Week_3_day_2_1107_52"></a>Week 3, day 2 (11/07)</h3>
<ul class="bullets">
<li>Made LB+CAM plates</li>
<li>Plated and spread 50µL and 100µL DH5α cells onto LB agar plates using sterile loops.
<ul class="bullets">
<li>Placed in overnight incubator</li>
</ul>
</li>
</ul>
<h3><a id="Week_3_day_3_1207_57"></a>Week 3, day 3 (12/07)</h3>
<p>-Checked on DH5α plates from the incubator.</p>
<h3><a id="Week_3_day_4_1307_62"></a>Week 3, day 4 (13/07)</h3>
<ul class="bullets">
<li>KAN and CAM antibiotic stocks</li>
<li>500mL each of LB+CAM and LB+KAN</li>
<li>500mL 1X TAE buffer from 50X TAE buffer and MilliQ water</li>
<li>Prepared overnight culture of one DH5α colony in 5mL LB at 37°C.</li>
</ul>
<h3><a id="Week_3_day_5_1407_68"></a>Week 3, day 5 (14/07)</h3>
<ul class="bullets">
<li>Checked on DH5α culture
<ul class="bullets">
<li>Initial OD600 was 0.011, so waited 110min for optimal OD600</li>
<li>Moved culture to ice once OD600 reached 0.466</li>
<li>Made DH5α stocks stored in -80°C, competence to be tested on Monday using iGEM competence cell test kit.</li>
</ul>
</li>
</ul>
<h3><a id="Week_4_day_1_1707_74"></a>Week 4, day 1 (17/07)</h3>
<ul class="bullets">
<li>Plated transformed cells.
<ul class="bullets">
<li>Spread the DH5α cells onto 4 LB+CAM plates, each containing 100µL of the 100pg/µL, 50pg/µL, 10pg/µL, and 0pg/µL cells.</li>
<li>Spread the DH10β cells onto two LB+KAN plates, both containing 100µL of the control and either 100µL or 50µL of the pKDL071 cells</li>
</ul>
</li>
<li>Incubated the plates at 37°C overnight.</li>
</ul>
<h3><a id="Week_4_day_2_1807_80"></a>Week 4, day 2 (18/07)</h3>
<ul class="bullets">
<li>Made LB media</li>
<li>Made LB+CAM plates</li>
<li>Checked on 17/07 transformed cells and moved them to the 4°C fridge.
<ul class="bullets">
<li>Confirmed DH5α cells were competent</li>
<li>Note: LB+CAM plates made on 13/07 may have been made with too much CAM (1250µL CAM to 500mL LB). This may explain why there were so few colonies from 17/07 transformation of DH5α + RFP/pSB1C3.</li>
</ul>
</li>
</ul>
<ul class="bullets">
<li>Obtained single colonies from transformed DH5α and DH10β plates from 17/07 and made overnight 37°C colonies.
<ul class="bullets">
<li>The DH5α was placed in the 5mL of LB+CAM (5µL of stock CAM added to 5mL LB).</li>
<li>The DH10β was placed in 5mL of LB+KAN (25µL of stock KAN added to 5mL of LB).</li>
</ul>
</li>
</ul>
<h3><a id="Week_4_day_3_1907_96"></a>Week 4, day 3 (19/07)</h3>
<ul class="bullets">
<li>Examined plates from 18/07
<ul class="bullets">
<li>Most plates had no cells likely due to the high CAM concentration (2.5x working concentration).</li>
</ul>
</li>
</ul>
<ul class="bullets">
<li>Made glycerol stocks of DH10β with pKDL071</li>
</ul>
<h3><a id="Week_4_day_5_2107_104"></a>Week 4, day 5 (21/07)</h3>
<ul class="bullets">
<li>Made LB+AMP plates</li>
</ul>
<h3><a id="Week_5_day_1_2407_107"></a>Week 5, day 1 (24/07)</h3>
<ul class="bullets">
<li>Made LB+CAM plates</li>
</ul>
<h3><a id="Week_5_day_3_2607_110"></a>Week 5, day 3 (26/07)</h3>
<ul class="bullets">
<li>Made LB+CAM and LB+KAN solutions for overnight cultures</li>
</ul>
<h3><a id="Week_5_day_4_2607_113"></a>Week 5, day 4 (26/07)</h3>
<ul class="bullets">
<li>Made 17 LB+KAN plates</li>
</ul>
<h3><a id="Week_5_day_5_2807_116"></a>Week 5, day 5 (28/07)</h3>
<ul class="bullets">
<li>Prepared 10X PBS buffer</li>
</ul>
<h3><a id="Week_6_day_1_3107_119"></a>Week 6, day 1 (31/07)</h3>
<ul class="bullets">
<li>Made 50mL non-sterilized 1X PBS Buffer</li>
</ul>
<h3><a id="Week_6_day_3_0208_122"></a>Week 6, day 3 (02/08)</h3>
<ul class="bullets">
<li>Made LB+CAM plates</li>
</ul>
<h3><a id="Week_6_day_5_0408_125"></a>Week 6, day 5 (04/08)</h3>
<ul class="bullets">
<li>Made LB liquid media</li>
</ul>
<h3><a id="Week_7_day_4_1008_128"></a>Week 7, day 4 (10/08)</h3>
<ul class="bullets">
<li>Made 17 LB+Kan plates</li>
</ul>
<h3><a id="Week_9_day_2_2208_131"></a>Week 9, day 2 (22/08)</h3>
<ul class="bullets">
<li>Made LB+KAN plates</li>
</ul>
<h3><a id="Week_9_day_3_2308_134"></a>Week 9, day 3 (23/08)</h3>
<ul class="bullets">
<li>Made LB+AMP plates</li>
</ul>
<h3><a id="Week_9_day_4_2408_137"></a>Week 9, day 4 (24/08)</h3>
<ul class="bullets">
<li>Made 5mL LB+Kan in falcon tube</li>
</ul>
<h3><a id="Week_10_day_4_0109_140"></a>Week 10, day 4 (01/09)</h3>
<ul class="bullets">
<li>Made LB+CAM plates</li>
</ul>
<h3><a id="Week_16_day_5_0610_143"></a>Week 16, day 5 (06/10)</h3>
<ul class="bullets">
<li>Made LB+AMP plates</li>
</ul>
<h3><a id="Week_18_day_2_1710_146"></a>Week 18, day 2 (17/10)</h3>
<ul class="bullets">
<li>Made LB+KAN and LB+CAM plates</li>
</ul>
<h3><a id="Week_19_day_7_2910_149"></a>Week 19, day 7 (29/10)</h3>
<ul class="bullets">
<li>Made 20% stock D-glucose solution by adding 20g of glucose to 100ml MilliQ water then autoclaved</li>
<li>Made Modified M9 Media (+0.2% glucose + 0.2% arabinose)</li>
</ul>
</div>
<div id='interlab_152' class='subsection'><h2 class="text-red"><a id="Interlab_152"></a>Interlab</h2>
<h3><a id="Week_3_day_4_1307_153"></a>Week 3, day 4 (13/07)</h3>
<ul class="bullets">
<li>Prepared overnight culture of one DH5α colony in 5mL LB at 37°C</li>
</ul>
<h3><a id="Week_3_day_5_1407_156"></a>Week 3, day 5 (14/07)</h3>
<ul class="bullets">
<li>Checked on DH5α culture
<ul class="bullets">
<li>Initial OD600 was 0.011, so waited 110min for optimal OD600</li>
<li>Moved culture to ice once OD600 reached 0.466</li>
<li>Made DH5α stocks stored in -80°C, competence to be tested on Monday using iGEM competence cell test kit.</li>
</ul>
</li>
</ul>
<h3><a id="Week_4_day_1_1707_162"></a>Week 4, day 1 (17/07)</h3>
<ul class="bullets">
<li>Transformed DH5α cells using iGEM competence cell test kit with RFP/pSB1C3 at concentrations 100pg/µL, 50pg/µL, 10pg/µL, and 0pg/µL (control)</li>
<li>Plated the cells onto 4 LB+CAM plates, each containing 100µL of the 100pg/µL, 50pg/µL, 10pg/µL, and 0pg/µL cells.</li>
<li>Incubated the plates at 37°C overnight.</li>
</ul>
<h3><a id="Week_4_day_2_1807_167"></a>Week 4, day 2 (18/07)</h3>
<ul class="bullets">
<li>Checked on transformed cells from previous day and moved them to the 4°C fridge.
<ul class="bullets">
<li>Confirmed DH5α cells were competent</li>
<li>Note: LB+CAM plates made on 13/07 may have been made with too much CAM (1250µL CAM to 500mL LB). This may explain why there were so few colonies from 17/07 transformation of DH5α + RFP/pSB1C3.</li>
</ul>
</li>
</ul>
<ul class="bullets">
<li>Resuspended the iGEM kit parts containing the DNA required for the Interlab study (DNA was obtained from kit plate #6, which was then moved to the 4°C fridge for storage) and transformed the chemically competent DH5α cells (from 14/07) with resuspended DNA
<ul class="bullets">
<li>Made 9 tubes: 8 GFP samples and one no plasmid control (labelled 20B, 20D, 20F, 20H, 20J,  20L, 20N, and 20P, respectively).</li>
<li>Originally, the goal was to make 17 tubes for two transformations (8 samples x 2 + 1 no plasmid control), however, there were not enough cells to to make a second set (missing second set of 20L and 20B).</li>
<li>Spread 100µL of transformed bacteria on LB+CAM plates to incubate and dry overnight at 37°C.</li>
</ul>
</li>
</ul>
<h3><a id="Week_4_day_3_1907_182"></a>Week 4, day 3 (19/07)</h3>
<ul class="bullets">
<li>Examined plates from yesterday
<ul class="bullets">
<li>Most plates had no cells likely due to the high CAM concentration (2.5x working concentration).</li>
</ul>
</li>
</ul>
<p>07-19 plates</p>
<ul class="bullets">
<li>Re-plated DH5α cells transformed with Interlab study plasmids from transformation tubes done on Tuesday and incubated overnight in 37°C shaker.</li>
</ul>
<h3><a id="Week_4_day_4_2007_190"></a>Week 4, day 4 (20/07)</h3>
<ul class="bullets">
<li>Obtained 19/07 GFP plates from the 37°C shaker.</li>
<li>Picked a single colony from each plate and incubated in 1.5mL LB media overnight in 37°C shaker and left to grow in the 37 degree C shaker.</li>
<li>In total, there were 8 sample tubes containing the GFP interlab samples and one control tube of just LB.</li>
<li>Made 5mL cultures of the DH5α cells transformed with GFP plasmids and placed them in the overnight shaker to grow.</li>
</ul>
<h3><a id="Week_4_day_5_2107_196"></a>Week 4, day 5 (21/07)</h3>
<ul class="bullets">
<li>Plated transfromed DH5a</li>
</ul>
<h3><a id="Week_4_day_6_2207_199"></a>Week 4, day 6 (22/07)</h3>
<ul class="bullets">
<li>Checked plates of DH5a cells transformed with Interlab plasmids (from 21/07).</li>
<li>Colonies grew into a lawn of bacteria and no single colonies could be isolated.</li>
</ul>
<h3><a id="Week_5_day_2_2507_205"></a>Week 5, day 2 (25/07)</h3>
<ul class="bullets">
<li>Obtained single colonies from the GFP sample plates (21/07) and plated them on LB+CAM plates. The plates were left at 37°C to grow overnight.</li>
</ul>
<h3><a id="Week_5_day_3_2607_208"></a>Week 5, day 3 (26/07)</h3>
<ul class="bullets">
<li>Examined LB+CAM plates and saw single colonies</li>
</ul>
<ul class="bullets">
<li>Inoculated single colonies from plates into LB+KAN and LB+CAM and put in incubator</li>
</ul>
<h3><a id="Week_5_day_4_2607_215"></a>Week 5, day 4 (26/07)</h3>
<ul class="bullets">
<li>Made 80x1mL glycerol stocks of 26/07 cultures and stored in -80°C freezer</li>
</ul>
<h3><a id="Week_5_day_5_2807_218"></a>Week 5, day 5 (28/07)</h3>
<ul class="bullets">
<li>Ran LUDOX-S40 calibration</li>
</ul>
<p>Table 1: LUDOX Calibration</p>
<table class="table table-striped table-bordered">
<thead>
<tr>
<th>100% LUDOX</th>
<th>H&lt;sub&gt;2&lt;/sub&gt;O</th>
</tr>
</thead>
<tbody>
<tr>
<td>0.0508</td>
<td>0.0402</td>
</tr>
<tr>
<td>0.0561</td>
<td>0.0415</td>
</tr>
<tr>
<td>0.0525</td>
<td>0.0419</td>
</tr>
<tr>
<td>0.0527</td>
<td>0.0435</td>
</tr>
</tbody>
</table>
<h3><a id="Week_6_day_1_3107_230"></a>Week 6, day 1 (31/07)</h3>
<ul class="bullets">
<li>Transferred samples 20J, 20L, 20N, and 20P from glycerol aliquots (from 26/07) to a 96-well plate and measured maximum emission and exciation wavelengths of GFP and YFP.
<ul class="bullets">
<li>GFP: Literature said the maximum exictation wavelength is 395 nm, the maximum emission wavelength is 509 nm
<ul class="bullets">
<li>We measured maximum exication: 390 nm, maximum emission: 400-750 nm.</li>
</ul>
</li>
<li>YFP: Literature said the maximum excitation wavelength is 512 nm, the maximum emission wavelength is 529 nm.</li>
<li><em>Note: No negative control was measured so data can’t be normalized.</em></li>
</ul>
</li>
</ul>
<h3><a id="Week_6_day_2_0108_237"></a>Week 6, day 2 (01/08)</h3>
<ul class="bullets">
<li>Measured the emission spectra of GFP samples 20B, 20D, and 20F with the plate reader</li>
<li>The interlab GFP samples use the mut3b GFP, which has a shifted excitation and emission spectra
<ul class="bullets">
<li>Excitation range of 488 nm - 501 nm</li>
<li>Emission maximum at 511 nm</li>
<li>At a 395 nm excitation wavelength, this GFP mutant emits 0.2x the intensity of wild type GFP. However, at a excitation wavelength of 488 nm, the emission intensity is 21x that of wild type GFP.</li>
</ul>
</li>
<li>The emission spectra was tested using samples 20B (positive control), 20D (negative control), and 20F from glycerol aliquots.
<ul class="bullets">
<li>In total, three different excitation spectra were tested: 488 nm, 508 nm, and 495 nm.</li>
</ul>
</li>
</ul>
<h3><a id="Week_6_day_3_0208_246"></a>Week 6, day 3 (02/08)</h3>
<ul class="bullets">
<li>Washed glycerol stocks and then suspended in LB+CAM to ensure consistent measurements from the plate reader
<ul class="bullets">
<li>All subsequent interlab samples will be measured in LB+CAM</li>
</ul>
</li>
<li>Restreaked samples to obtain biological replicates</li>
<li>Measured excitation spectra using plate reader in WB 316 of GFP samples 20B, 20D, and 20F.</li>
</ul>
<h3><a id="Week_6_day_5_0408_252"></a>Week 6, day 5 (04/08)</h3>
<ul class="bullets">
<li>Restreaked 20D transfomed cell</li>
</ul>
<ul class="bullets">
<li>Took plate reader measurements according to Interlab protocol
<ul class="bullets">
<li>Used LUDOX 100% and H2O to set the standard point</li>
<li>Found fluorescein fluorescence standard curve</li>
<li>Took measurements of all cell samples</li>
</ul>
</li>
</ul>
</div>
<div id='LacILOV' class='subsection'><h2 class="text-red"><a id="LacILOV_263"></a>LacILOV</h2>
<h3><a id="Week_1_day_5_3006_264"></a>Week 1, day 5 (30/06)</h3>
<ul class="bullets">
<li>Bacterial transformation of competent cells with RFP plasmid</li>
</ul>
<h3><a id="Week_2_day_5_0707_267"></a>Week 2, day 5 (07/07)</h3>
<ul class="bullets">
<li>Electroporation of MG1655ΔLacI electrocompetent cells.</li>
<li>Modified electroporation protocol to incubate transformed cells at 300rpm, 37°C after SOC recovery step.</li>
<li>Bacterial transformation of electroporated cells with RFP plasmid (BBa_j04450 in pSB1C3) to check transformation efficiency at 0, 10, 50 and 100pg/ul.</li>
<li>MiniPrepped transformed DH10β (BBa_j04450 in pSB1C3) to purify plasmid (sample 1/2 stored in -20°C)</li>
<li>Performed RE digestion, nicked with EcoRI-HF (sample 1/2 stored in -20°C)</li>
</ul>
<h3><a id="Week_3_day_1_1007_274"></a>Week 3, day 1 (10/07)</h3>
<ul class="bullets">
<li>Miniprepped RFP plasmids on the pSB1C3 backbone from chemically competent DH10β cells from 07/07.</li>
<li>Conducted gel electrophoresis of RE digested and non RE digested dupicate samples from 07/07.
<ul class="bullets">
<li>10µL of each ladder was used, but in the future we should experiment with adding less ladder (conserve reagents!).</li>
<li>Each 10µg sample was mixed with 2µL loading dye and loaded on the gel along with two ladders. The sequences are as follows:
<ol>
<li>EcoRI-HF Digested Sample 1 (D1)</li>
<li>Undigested Sample 1 (UD1)</li>
<li>1kb DNA ladder for measurement</li>
<li>EcoRI-HF Digested Sample 2 (D2)</li>
<li>Undigested Sample 2 (UD2)</li>
<li>1kb DNA ladder for measurement</li>
</ol>
</li>
</ul>
</li>
<li>Demonstration: Nanodropped each sample to show the protocol to general members.</li>
</ul>
<h3><a id="Week_3_day_2_1107_289"></a>Week 3, day 2 (11/07)</h3>
<p>Calculated optimization for Q5 (High Fidelity) PCR of UNS3/2 into ends of  pSB1C3 backbone</p>
<ul class="bullets">
<li>Optimized annealing step because our primer Tms are high (85°C), so using different additives/temperature gradients to optimize yield</li>
</ul>
<p>“07-11 Tm Calculator.png”</p>
<p>Table 1: PCR Optimization</p>
<table class="table table-striped table-bordered">
<thead>
<tr>
<th>Component</th>
<th>No treatment (µL)</th>
<th>3% DMSO (µL)</th>
<th>6% DMSO (µL)</th>
<th>1X GC Buffer (µL)</th>
<th>No DNA control (µL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>5X Q5 Reaction Buffer</td>
<td>5</td>
<td>5</td>
<td>5</td>
<td>5</td>
<td>5</td>
</tr>
<tr>
<td>10mM dNTPs</td>
<td>0.5</td>
<td>0.5</td>
<td>0.5</td>
<td>0.5</td>
<td>0.5</td>
</tr>
<tr>
<td>10µM Forward primers</td>
<td>1.25</td>
<td>1.25</td>
<td>1.25</td>
<td>1.25</td>
<td>1.25</td>
</tr>
<tr>
<td>10µM Reverse primers</td>
<td>1.25</td>
<td>1.25</td>
<td>1.25</td>
<td>1.25</td>
<td>1.25</td>
</tr>
<tr>
<td>1ng/µL Template DNA (prediluted)</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>0</td>
</tr>
<tr>
<td>Q5 HF DNA Polymerase</td>
<td>0.25</td>
<td>0.25</td>
<td>0.25</td>
<td>0.25</td>
<td>0.25</td>
</tr>
<tr>
<td>1X GC Buffer</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>5</td>
<td>5</td>
</tr>
<tr>
<td>DMSO</td>
<td>0</td>
<td>0.75</td>
<td>1.5</td>
<td>0</td>
<td>0</td>
</tr>
<tr>
<td>Nuclease-free water</td>
<td>15.75</td>
<td>15</td>
<td>14.25</td>
<td>10.75</td>
<td>11.75</td>
</tr>
<tr>
<td>Total volume</td>
<td>25</td>
<td>25</td>
<td>25</td>
<td>25</td>
<td>25</td>
</tr>
</tbody>
</table>
<p>* Multiply five for 5 reactions (5 different annealing temperatures: 72, 71, 70, 69, and 68°C).</p>
<p>Table 2: Master Mixes</p>
<table class="table table-striped table-bordered">
<thead>
<tr>
<th>Master Mix</th>
<th>No treatment (µL)</th>
<th>3% DMSO (µL)</th>
<th>6% DMSO (µL)</th>
<th>1X GC Buffer (µL)</th>
<th>No DNA control (µL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>GC Buffer</td>
<td>0</td>
<td>0</td>
<td>0</td>
<td>40</td>
<td>40</td>
</tr>
<tr>
<td>DMSO</td>
<td>0</td>
<td>6</td>
<td>12</td>
<td>0</td>
<td>0</td>
</tr>
<tr>
<td>Nuclease-free water</td>
<td>126</td>
<td>120</td>
<td>114</td>
<td>86</td>
<td>94</td>
</tr>
<tr>
<td>Sum</td>
<td>126</td>
<td>126</td>
<td>126</td>
<td>126</td>
<td>134</td>
</tr>
</tbody>
</table>

<h3><a id="Week_3_day_3_1207_336"></a>Week 3, day 3 (12/07)</h3>
<p>Performed Q5 HF PCR (discussed 11/07) with modifications:</p>
<ul class="bullets">
<li>Revised PCR calculations based on Christian’s suggestions: 7 Q5 HF PCR 50µL reactions with 3% DMSO in each sample (one control tube with no DNA template).</li>
</ul>
<p>Table 1: Revised PCR Calculations</p>
<table class="table table-striped table-bordered">
<thead>
<tr>
<th>Component</th>
<th>Reaction (×7)</th>
<th>Control</th>
</tr>
</thead>
<tbody>
<tr>
<td>5X Q5 Reaction Buffer</td>
<td>10</td>
<td>10</td>
</tr>
<tr>
<td>10mM dNTPs</td>
<td>1</td>
<td>1</td>
</tr>
<tr>
<td>10µM Forward primers</td>
<td>2.5</td>
<td>2.5</td>
</tr>
<tr>
<td>10µM Reverse primers</td>
<td>2.5</td>
<td>2.5</td>
</tr>
<tr>
<td>1ng/µL Template DNA (prediluted)</td>
<td>1</td>
<td>0</td>
</tr>
<tr>
<td>Q5 HF DNA Polymerase</td>
<td>0.5</td>
<td>0.5</td>
</tr>
<tr>
<td>1X GC Buffer</td>
<td>10</td>
<td>10</td>
</tr>
<tr>
<td>100% DMSO</td>
<td>1.5</td>
<td>1.5</td>
</tr>
<tr>
<td>Nuclease-free water</td>
<td>31</td>
<td>32</td>
</tr>
</tbody>
</table>
<ul class="bullets">
<li>Prepared master mix for Q5 HF PCR reactions (with 50µL 3% DMSO). Template DNA is included in 6 tubes with 1 control without template DNA.
<ul class="bullets">
<li>One template DNA tube only contains 25 uL, possibly due to pippetting error
<ul class="bullets">
<li>Note: prepare a master mix of 8 reactions of 50 μL, instead of 7.1 reactions, to accommodate pippeting errors).</li>
</ul>
</li>
</ul>
</li>
<li>Ran PCR. Tubes were meant to be optimized through a temperature gradient, but were placed incorrectly in the PCR machine, so they were run all at the same temperature.</li>
<li>Ran a gel electrophoresis of psB1C3 backbone PCR. The gel was not successful. (Life: What a cruel mistress)
<ul class="bullets">
<li>Note: Gel was made with 15 wells - too small for 10µl samples (15 wells should use ~6-8µl).</li>
</ul>
</li>
</ul>
<h3><a id="Week_3_day_4_1307_366"></a>Week 3, day 4 (13/07)</h3>
<p>Performed Q5 HF PCR to amplify pSB1C3 backbone</p>
<ul class="bullets">
<li>Prepared same master master mixes as calculated 12/07 for 6 samples with template DNA and one control without template DNA</li>
<li>Ran the PCR of samples 1-6 at different temperatures, one each at 72.0, 71.4, 70.1, 68.3, 66, and 64.3°C respectively
<ul class="bullets">
<li>Control was tested at 70.1°C</li>
</ul>
</li>
<li>Performed gel electrophoresis of Q5 PCR samples, along with EcoRI digested pSB1C3 with RFP control</li>
</ul>
<ul class="bullets">
<li>Advisor suggested results may be due to primer dimerization, possibly because of high primer Tm</li>
<li>Planned to redesign primers to reduce this problem by removing base pairs from the 3’ binding up to ~15bp homology between primers and backbone
<ul class="bullets">
<li>Aiming for Tm of ~68°C</li>
</ul>
</li>
</ul>
<h3><a id="Week_3_day_5_1407_378"></a>Week 3, day 5 (14/07)</h3>
<p>Redesigned primers based on suggestions from 13/07 to have lower Tm</p>
<ul class="bullets">
<li>New reverse primer for pKDL071 backbone</li>
<li>New forward and reverse primers for pSB1C3 backbone</li>
</ul>
<p>The Q5 polymerase used in previous PCRs is very bubbly which could have been a source of error, so instead we performed 2-step NEB Phusion HF PCR of pSB1C3 with RFP construct</p>
<ul class="bullets">
<li>Used primers from 2016 and 2017 to compare results</li>
<li>Tubes were labelled 16.1-16.5 (for 2016 primers) and 17.1-17.5 (for 2017 primers)</li>
<li>Controls without DNA template were labelled NTC (no template control)</li>
<li>Ran the samples on a temperature gradient to determine the optimal PCR settings</li>
</ul>
<p>Table 1: Phusion PCR Reaction Component Table (for total volume of 50µL per tube)</p>
<p>| Component | 16.NTC | 16.1 | 16.2 | 16.3 | 16.4 | 16.5 | 17.NTC | 17.1 | 17.2 | 17.3 | 17.4 | 17.5 |<br>
| — | — | — | — | — | — | — | — | — | — | — | — |<br>
| 5X Phusion HF or GC Buffer | 10µL | 10µL | 10µL | 10µL | 10µL | 10µL | 10µL | 10µL | 10µL | 10µL | 10µL | 10µL |<br>
| 10mM dNTPs | 1µL | 1µL | 1µL | 1µL | 1µL | 1µL | 1µL | 1µL | 1µL | 1µL | 1µL | 1µL |<br>
| 10µM 2016 Forward Primer | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 0µL | 0µL | 0µL | 0µL | 0µL | 0µL |<br>
| 10µM 2016 Reverse Primer | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 0µL | 0µL | 0µL | 0µL | 0µL | 0µL |<br>
| 10µM 2017 Forward Primer | 0µL | 0µL | 0µL | 0µL | 0µL | 0µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL |<br>
| 10µM 2017 Reverse Primer | 0µL | 0µL | 0µL | 0µL | 0µL | 0µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL | 2.5µL |<br>
| Template DNA | 0µL | 1µL | 1µL | 1µL | 1µL | 1µL | 0µL | 1µL | 1µL | 1µL | 1µL | 1µL |<br>
| DMSO | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL | 1.5µL |<br>
| Phusion DNA Polymerase | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL | 0.5µL |<br>
| Nuclease-free water | 32µL | 31µL | 31µL | 31µL | 31µL | 31µL | 32µL | 31µL | 31µL | 31µL | 31µL | 31µL |</p>
<p>Table 2: Phusion PCR Temperature Gradient</p>
<table class="table table-striped table-bordered">
<thead>
<tr>
<th>Temperature</th>
<th>Tubes</th>
</tr>
</thead>
<tbody>
<tr>
<td>62°C</td>
<td>16.1, 17.1</td>
</tr>
<tr>
<td>63.7°C</td>
<td>16.2, 17.2</td>
</tr>
<tr>
<td>68°C</td>
<td>16.3, 16.NTC, 17.3, 17.NTC</td>
</tr>
<tr>
<td>70.5°C</td>
<td>16.4, 17.4</td>
</tr>
<tr>
<td>72°C</td>
<td>16.5, 17.5</td>
</tr>
</tbody>
</table>
<ul class="bullets">
<li>Performed gel electrophoresis of Phusion PCR samples</li>
</ul>
<p>“07-14 gel of Phusion PCR”</p>
<h3><a id="Week_4_day_1_1707_424"></a>Week 4, day 1 (17/07)</h3>
<ul class="bullets">
<li>Performed bacterial transformation
<ul class="bullets">
<li>Transformed DH5α cells using iGEM competence cell test kit with pSB1C3+RFP at concentrations 100pg/µL, 50pg/µL, 10pg/µL, and 0pg/µL (control)</li>
<li>Transformed DH10β cells with pKDL071 or no plasmid control</li>
<li>Plated transformed cells.
<ul class="bullets">
<li>Spread the DH5α cells onto 4 LB+CAM plates, each containing 100µL of the 100pg/µL, 50pg/µL, 10pg/µL, and 0pg/µL cells.</li>
<li>Spread the DH10β cells onto two LB+KAN plates, both containing 100µL of the control and either 100µL or 50µL of the pKDL071 cells</li>
</ul>
</li>
<li>Incubated the plates at 37°C overnight.</li>
</ul>
</li>
</ul>
<h3><a id="Week_4_day_2_1807_433"></a>Week 4, day 2 (18/07)</h3>
<ul class="bullets">
<li>Checked on 17/07 transformed cells and moved them to the 4°C fridge.
<ul class="bullets">
<li>Confirmed DH5α cells were competent</li>
<li>Note: LB+CAM plates made on 13/07 may have been made with too much CAM (1250µL CAM to 500mL LB). This may explain why there were so few colonies from 17/07 transformation of DH5α + RFP/pSB1C3.</li>
</ul>
</li>
</ul>
<ul class="bullets">
<li>Obtained single colonies from DH5α and DH10β plates from 17/07 and made overnight 37°C colonies.
<ul class="bullets">
<li>The DH5α was placed in the 5mL of LB+CAM (5µL of stock CAM added to 5mL LB).</li>
<li>The DH10β was placed in 5mL of LB+KAN (25µL of stock KAN added to 5mL of LB).</li>
</ul>
</li>
</ul>
<h3><a id="Week_4_day_3_1907_447"></a>Week 4, day 3 (19/07)</h3>
<ul class="bullets">
<li>Examined plates from 18/07
<ul class="bullets">
<li>Most plates had no cells likely due to the high CAM concentration (2.5x working concentration).</li>
</ul>
</li>
</ul>
<ul class="bullets">
<li>Miniprepped DH10β and DH5α to extract plasmides (pKDL071 and pSB1C3 with RFP respectively) and did Nanodrop on the results</li>
</ul>
<p>Table 1: Nanodrop results from Miniprep</p>
<table class="table table-striped table-bordered">
<thead>
<tr>
<th>Plasmid</th>
<th>260/280</th>
<th>260/230</th>
<th>ng/µL</th>
</tr>
</thead>
<tbody>
<tr>
<td>pSB1C3 #1</td>
<td>1.82</td>
<td>1.42</td>
<td>86.6</td>
</tr>
<tr>
<td>pSB1C3 #2</td>
<td>1.85</td>
<td>1.52</td>
<td>87.9</td>
</tr>
<tr>
<td>psB1C3 #3</td>
<td>1.79</td>
<td>0.99</td>
<td>136.6</td>
</tr>
<tr>
<td>pKDL071 #1</td>
<td>1.89</td>
<td>1.55</td>
<td>60.0</td>
</tr>
<tr>
<td>pKDL071 #2</td>
<td>1.78</td>
<td>1.25</td>
<td>66.0</td>
</tr>
<tr>
<td>pKDL071 #3</td>
<td>1.95</td>
<td>1.90</td>
<td>48.7</td>
</tr>
</tbody>
</table>
<ul class="bullets">
<li>Performed Phusion PCR of pSB1C3 with RFP construct using psB1C3 #2 and the pKDL071 FWD/RVS primers
<ol>
<li>Made 1ng/uL of pDKL071 #3 for PCR via following steps:</li>
</ol>
<ul class="bullets">
<li>2μL of pDKL071 #3 Miniprep was mixed with 95.4 μL of nuclease free water</li>
<li>Made 100μL aliquots of 100μM from pKDL071 FWD UNS3 primer (shipped 05/07) resuspended in 785μL nuclease free water</li>
<li>Made 100μL aliquots of 100μM from pKDL071 REV UNS2 primer (shipped 05/07) resuspended in 761μL nuclease free water</li>
</ul>
<ol start="2">
<li>Made a Master mix without DNA template containing:</li>
</ol>
<ul class="bullets">
<li>17.5μL of 10 uM FWD_UNS3 primer</li>
<li>17.5μL of 10 uM REV_UNS2 primer</li>
<li>70uL of 5x phusion buffer</li>
<li>7uL of 10mM dNTPS</li>
<li>10.5uL 100% DMSO ( to reach a final concentration of 3%)</li>
<li>3.5uL Phusion polymerase</li>
</ul>
<ol start="3">
<li>Took 18uL was taken from this master mix added 32 uL of Nuclease free water for a final volume of 50 uL as the No Template Control (NTC)</li>
<li>Added DNA template to the Master Mix:</li>
</ol>
<ul class="bullets">
<li>6 uL 1ng/uL Miniprepped pKDL071 #3</li>
<li>186 uL nuclease free water</li>
</ul>
<ol start="5">
<li>Added 50uL of the Master Mix to 5 PCR tubes</li>
<li>Ran Phusion PCR on temperature gradient with samples 1-5 at 62, 63.7, 68, 71.3, and 72°C respectively</li>
</ol>
<ul class="bullets">
<li>NTC was at 72°C</li>
</ul>
</li>
<li>Ran gel on 12μL samples of Phusion PCR results for 1 minute then left for 1 hour then run for 1 hour due to equipment issues.
<ul class="bullets">
<li>Timing issues due might be why the ladder is not as intense.</li>
<li>Sample 3 appears to be our optimal temperature @ 68°C</li>
</ul>
</li>
</ul>
<h3><a id="Week_4_day_4_2007_491"></a>Week 4, day 4 (20/07)</h3>
<p>Ran Phusion PCR of the pKDL071 plasmid with RFP using the pKDL071 plasmid from Miniprep #3 (19/07).</p>
<ul class="bullets">
<li>Used the pKDL071 FWD/RVS primers shipped on 05/07 (the first set of primers designed in 2017)</li>
<li>Made Master Mix without DNA template or primers (for 8 reactions)
<ul class="bullets">
<li>20 μL of 10 uM FWD_UNS3 primer</li>
<li>20 μL of 10 uM REV_UNS2 primer</li>
<li>80 uL of 5x phusion buffer</li>
<li>8 uL of 10mM dNTPS</li>
<li>12 uL 100% DMSO ( to reach a final concentration of 3%)</li>
<li>4 uL Phusion polymerase</li>
</ul>
</li>
<li>18 uL was taken from the master mix as the No Template Control (NTC) and mixed with 32 uL of Nuclease free water.</li>
<li>Added DNA template to Master Mix (for 7 reactions)
<ul class="bullets">
<li>7 uL of 1ng/uL miniprep#3 pKDL071</li>
<li>217 uL of nuclease free water</li>
</ul>
</li>
<li>50 uL of the master mix were added to 5 PCR tubes</li>
<li>6 samples and one NTC run at 68°C, the optimal temperature determined on 17/07.</li>
</ul>
<p>Optimization Phusion PCRs of pSB1C3 and pKDL071 plasmids with new UNS primers</p>
<ul class="bullets">
<li>Made 5 samples and one NTC for PCR of the pSB1C3 plasmid using pSB1C3 miniprep#2 and the new forward and reverse UNS primers.
<ul class="bullets">
<li>Annealing temperature was 64°C so samples 1-5 at 72, 70.1, 66, 64.3, and 62°C respectively with NTC at 64.3°C</li>
</ul>
</li>
<li>Made 5 samples and one NTC for PCR of the pKDL071 plasmid using pKDL071 miniprep#3, the old forward, and new reverse UNS primers
<ul class="bullets">
<li>Annealing temperature was 62°C so samples 1-5 at 67, 65.1, 63.3, 61, and 57°C respectively with NTC at 63.3°C</li>
</ul>
</li>
<li>Made Master mixes with DNA
<ul class="bullets">
<li>OLD_FWD and NEW_REV pDKL071 (7 reactions)
<ul class="bullets">
<li>17.5 μL of 10 uM FWD primer</li>
<li>17.5 μL of 10 uM REV primer</li>
</ul>
</li>
<li>NEW_FWD_USN3 and NEW_REV_UNS2 pSB1C3  (7 reactions)
<ul class="bullets">
<li>17.5 μL of 10 uM FWD_UNS3 primer</li>
<li>17.5 μL of 10 uM REV_UNS2 primer</li>
</ul>
</li>
</ul>
</li>
<li>Made Master mix without DNA template or primers (for 15 reactions)
<ul class="bullets">
<li>150 uL of 5x phusion buffer</li>
<li>15 uL of 10mM dNTPS</li>
<li>22.5 uL 100% DMSO ( to reach a final concentration of 3%)</li>
<li>7.5 uL Phusion polymerase</li>
</ul>
</li>
<li>Added 91 uL of Master mix without DNA to the pSB1C3 &amp; pDKL071 primer master mixes</li>
<li>18 uL was taken from each primer master mix as the No Template Control (NTC) and added to 32 uL of Nuclease free water to reach a final volume of 50 uL.</li>
<li>Added DNA template to primer mixes:
<ul class="bullets">
<li>pSB1C3 master with DNA template (for 6 reactions):
<ul class="bullets">
<li>5.5 uL of 1ng/uL miniprep pSB1C3</li>
<li>170.5 uL of nuclease free water</li>
</ul>
</li>
<li>pDKL071 master with DNA template (for 6 reactions)
<ul class="bullets">
<li>5.5 uL of 1ng/uL miniprep pDKL071</li>
<li>170.5 uL of nuclease free water</li>
</ul>
</li>
</ul>
</li>
<li>50 uL of each master mix were added to 5 PCR tubes (10 tubes total + 2 NTCs)</li>
</ul>
<p>Performed gel electrophoresis all PCR results</p>
<ul class="bullets">
<li>8 wells for Phusion PCR of the pKDL071 plasmid</li>
</ul>
<ul class="bullets">
<li>15 wells for optimization PCR of pSB1C3 and pKDL071 plasmids
<ul class="bullets">
<li>Wells 1-6 contained pSB1C3 samples 1-5 and NTC</li>
<li>Well 7 contained the 10kb ladder</li>
<li>Wells 8-13 contained the pKDL071 samples 1-5 and NTC</li>
</ul>
</li>
</ul>
<p>Made 5mL cultures of the DH5 alpha cells transformed with GFP plasmids and placed them in the overnight shaker to grow.</p>
<h3><a id="Week_4_day_5_2107_550"></a>Week 4, day 5 (21/07)</h3>
<p>Nanodrop PCR products of pKDL071 from 20/07</p>
<ul class="bullets">
<li>Note: NOT purified</li>
</ul>
<p>Table 1: Nanodrop of pKDL071-UNS with old primers (at 68°C)</p>
<table class="table table-striped table-bordered">
<thead>
<tr>
<th>Sample</th>
<th>ng/uL</th>
<th>260/280</th>
<th>260/230</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>200.7</td>
<td>1.34</td>
<td>0.47</td>
</tr>
<tr>
<td>2</td>
<td>379.0</td>
<td>1.32</td>
<td>0.62</td>
</tr>
<tr>
<td>3</td>
<td>357.5</td>
<td>1.37</td>
<td>0.62</td>
</tr>
<tr>
<td>4</td>
<td>346.1</td>
<td>1.33</td>
<td>0.64</td>
</tr>
<tr>
<td>5</td>
<td>363.9</td>
<td>1.33</td>
<td>0.59</td>
</tr>
<tr>
<td>6</td>
<td>353.6</td>
<td>1.37</td>
<td>0.61</td>
</tr>
<tr>
<td>NTC</td>
<td>339.3</td>
<td>1.34</td>
<td>0.57</td>
</tr>
</tbody>
</table>
<p>Table 2: Nanodrop of pKDL071-UNS optimization with new REV primer (at 62-72°C)</p>
<table class="table table-striped table-bordered">
<thead>
<tr>
<th>Sample</th>
<th>ng/uL</th>
<th>260/280</th>
<th>260/230</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>436.9</td>
<td>1.42</td>
<td>0.74</td>
</tr>
<tr>
<td>2</td>
<td>437.1</td>
<td>1.46</td>
<td>0.72</td>
</tr>
<tr>
<td>3</td>
<td>443.0</td>
<td>1.41</td>
<td>0.72</td>
</tr>
<tr>
<td>4</td>
<td>446.7</td>
<td>1.42</td>
<td>0.73</td>
</tr>
<tr>
<td>5</td>
<td>434.9</td>
<td>1.43</td>
<td>0.71</td>
</tr>
<tr>
<td>NTC</td>
<td>436.3</td>
<td>1.43</td>
<td>0.71</td>
</tr>
</tbody>
</table>
<p>Dpn1 digest and purification of pKDL071 PCR product with new REV primer: 40uL digestion reaction each tube.</p>
<ul class="bullets">
<li>6 tubes in total, including 4 samples (D1-D4), 1 NTC and 1 positive control (miniprepped pKDL071)</li>
<li>Master Mix contained:
<ul class="bullets">
<li>32uL of pKDL071</li>
<li>4uL 10X CutSmart</li>
<li>4uL Dpn1</li>
</ul>
</li>
<li>Nanodropped the results</li>
</ul>
<p>Table 3: Nanodrop of the digested and purified pKDL071 PCR product with new REV primer</p>
<table class="table table-striped table-bordered">
<thead>
<tr>
<th>Sample</th>
<th>ng/ul</th>
<th>260/280</th>
<th>260/230</th>
</tr>
</thead>
<tbody>
<tr>
<td>D1</td>
<td>28.1</td>
<td>1.82</td>
<td>1.69</td>
</tr>
<tr>
<td>D2</td>
<td>27.9</td>
<td>1.84</td>
<td>1.84</td>
</tr>
<tr>
<td>D3</td>
<td>31.6</td>
<td>1.80</td>
<td>1.73</td>
</tr>
<tr>
<td>D4</td>
<td>30.5</td>
<td>1.94</td>
<td>1.84</td>
</tr>
<tr>
<td>NTC</td>
<td>18.3</td>
<td>1.79</td>
<td>1.43</td>
</tr>
<tr>
<td>Positive Control (minipreped pKDL071)</td>
<td>22.3</td>
<td>1.81</td>
<td>1.70</td>
</tr>
</tbody>
</table>
<p>Performed Phusion PCR of pSB1C3 with new FWD UNS &amp; REV UNS with temperature gradient</p>
<ul class="bullets">
<li>Made Primer Master mix (MM1) new FWD UNS3 &amp; New REV UNS2 pSB1C3 (for 7 reactions) containing:
<ul class="bullets">
<li>17.5 μL of 10 uM NEW FWD_UNS3 primer</li>
<li>17.5 μL of 10 uM NEW REV_UNS2 primer</li>
</ul>
</li>
<li>Made Master mix (MM2) without DNA template or primers:
<ul class="bullets">
<li>70 uL of 5x Phusion buffer</li>
<li>7 uL of 10mM dNTPS</li>
<li>10.5 uL 100% DMSO ( to reach a final concentration of 3%)</li>
<li>3.5 uL Phusion polymerase</li>
</ul>
</li>
<li>Added MM2 to MM1 to make a Primer + Ploymerase mix</li>
<li>Took 18uL from polymerase + primer mix as NTC and added 32 uL of nuclease free water for final volume of 50uL</li>
<li>Added the following to make a Primer + ploymerase + DNA Mix (for 6 reactions)
<ul class="bullets">
<li>6 uL of 1 ng/uL miniprep pSB1C3 (RFP)</li>
<li>186 uL of Nuclease free water</li>
</ul>
</li>
<li>Divided final mix into 6 tubes, 50 uL each</li>
<li>Ran PCR with S1-6 in 62.3, 63.7, 66.3, 69.5, 71.3, and 62°C respectively, with NTC in 69.5°C</li>
<li>Ran a gel of the results</li>
</ul>
<h3><a id="Week_5_day_1_2407_616"></a>Week 5, day 1 (24/07)</h3>
<ul class="bullets">
<li>Dpn1 digest of pSB1C3+UNS PCR from 21/07
<ul class="bullets">
<li>4 sample tubes and 1 control without Dpn1</li>
</ul>
</li>
<li>Gel electrophoresis of the results</li>
</ul>
<ul class="bullets">
<li>Note: The absence of bands could be due to multiple reasons; the product was lost during digestion and purification or there was not enough product initially.</li>
<li>Nanodrop of the purified samples was preformed to test concentration of DNA purified.</li>
</ul>
<p>Table 1: Nanodrop results of purified Dpn1 digestion of pSB1C3+UNS</p>
<table class="table table-striped table-bordered">
<thead>
<tr>
<th>Sample</th>
<th>260/280</th>
<th>260/230</th>
<th>ng/uL</th>
</tr>
</thead>
<tbody>
<tr>
<td>D1</td>
<td>1.88</td>
<td>1.43</td>
<td>10.6</td>
</tr>
<tr>
<td>D2</td>
<td>1.74</td>
<td>1.34</td>
<td>11.8</td>
</tr>
<tr>
<td>D3</td>
<td>1.91</td>
<td>0.75</td>
<td>8.5</td>
</tr>
<tr>
<td>D4</td>
<td>1.87</td>
<td>1.64</td>
<td>13.6</td>
</tr>
<tr>
<td>Undigested pSB1C3</td>
<td>1.33</td>
<td>1.16</td>
<td>6.2</td>
</tr>
</tbody>
</table>
<h3><a id="Week_5_day_2_2507_635"></a>Week 5, day 2 (25/07)</h3>
<p>Assembled reporter and mCherry constructs</p>
<ul class="bullets">
<li>Pooled the purified pKDL071+UNS product from 21/07 purification to obtain a greater amount of DNA necessary for assembly.
<ul class="bullets">
<li>The resulting concentration of product, measured via Nanodrop, was 38.2 ng/uL (Contamination values: 260/280 = 1.93; 260/230 = 1.77).</li>
</ul>
</li>
<li>Created assembly according to the HiFi DNA assembly protocol. The reporter and mCherry master mix values are listed in the table below.</li>
</ul>
<p>Table 1: HiFi DNA Assembly Calculations</p>
<table class="table table-striped table-bordered">
<thead>
<tr>
<th>Sample</th>
<th>Reporter</th>
<th>mCherry</th>
</tr>
</thead>
<tbody>
<tr>
<td>pKDL071+UNS</td>
<td>1.67uL</td>
<td>1.67uL</td>
</tr>
<tr>
<td>Reporter mCherry</td>
<td>10.81uL</td>
<td>N/A</td>
</tr>
<tr>
<td>Reporter YFP</td>
<td>9.7uL</td>
<td>N/A</td>
</tr>
<tr>
<td>mCherry</td>
<td>N/A</td>
<td>10uL</td>
</tr>
<tr>
<td>Total fragment volume</td>
<td>22.18uL</td>
<td>11.67uL</td>
</tr>
<tr>
<td>Buffer (HiFi MM)</td>
<td>22.18uL</td>
<td>11.67uL</td>
</tr>
<tr>
<td>Total Reaction Volume</td>
<td>44.36uL</td>
<td>23.34uL</td>
</tr>
</tbody>
</table>
<ul class="bullets">
<li>Transformed the assembly parts into the NEB 5α competent cells (came with the HiFi DNA assembly kit).</li>
</ul>
<p>Plated NEB 5α cells to grow overnight in the 37°C shaker.</p>
<ul class="bullets">
<li>12 LB+KAN plates(6 mCherry, 6 reporter)</li>
<li>4 LB+AMP plates (2 for the positive control transformation (pUC19), 2 for the positive assembly control)</li>
</ul>
<h3><a id="Week_5_day_3_2607_659"></a>Week 5, day 3 (26/07)</h3>
<ul class="bullets">
<li>Examined NEB 5α plates from 25/07 and saw single colonies</li>
</ul>
<ul class="bullets">
<li>Inoculated single colonies from pKDL071 mCherry and Reporter plates into LB+KAN and LB+CAM and put in incubator</li>
</ul>
<p>Ran Phusion PCR of pSB1C3 (minprep #2) with new primers</p>
<ul class="bullets">
<li>Made 5 samples and 1 NTC</li>
<li>Loaded samples qPCR machine at 68°C</li>
</ul>
<h3><a id="Week_5_day_4_2607_671"></a>Week 5, day 4 (26/07)</h3>
<ul class="bullets">
<li>Gel Electrophoresis of pSB1C3 Phusion PCR from 26/07
<ul class="bullets">
<li>Primer dimers &amp; no amplified product</li>
<li>NEB’s 2-Log-DNA Ladders were used</li>
</ul>
</li>
</ul>
<ul class="bullets">
<li>Miniprepped mCherry and reporter (YFP) constructs from our pKDL071 plasmid backbone from DH5a transformed bacteria (from 25/07) with subsequent nanodrop to measure concentration of DNA:
<ul class="bullets">
<li>9 minipreps in total for mCherry construct:
<ul class="bullets">
<li>6 from inoculation culture of colony A and B</li>
<li>3 from inoculation culture of colony C</li>
</ul>
</li>
<li>6 minipreps in total for Reporter construct:
<ul class="bullets">
<li>3 minipreps from inoculation culture of colony A</li>
<li>3 minipreps from inoculation culture of colony B</li>
</ul>
</li>
<li>Nuclease free water was added to resuspend DNA after vacufuging the miniprepped product (to eliminate elution buffer)</li>
</ul>
</li>
</ul>
<p>Table 1: Nanodrop results of mCherry and Reporter pKDL071 plasmids</p>
<table class="table table-striped table-bordered">
<thead>
<tr>
<th>Sample</th>
<th>DNA Concentration (ng/uL)</th>
<th>260/280</th>
<th>260/230</th>
<th>Nuclease-free water (uL)</th>
</tr>
</thead>
<tbody>
<tr>
<td>mCherry 1 (A)</td>
<td>20.0</td>
<td>1.59</td>
<td>1.30</td>
<td>22.8571428571</td>
</tr>
<tr>
<td>mCherry 2 (A)</td>
<td>36.6</td>
<td>1.78</td>
<td>1.74</td>
<td>41.8285714286</td>
</tr>
<tr>
<td>mCherry 3 (A)</td>
<td>38.7</td>
<td>1.60</td>
<td>0.90</td>
<td>44.2285714286</td>
</tr>
<tr>
<td>mCherry 1 (B)</td>
<td>74.5</td>
<td>1.54</td>
<td>0.74</td>
<td>85.1428571429</td>
</tr>
<tr>
<td>mCherry 2 (B)</td>
<td>25.0</td>
<td>1.69</td>
<td>1.54</td>
<td>28.5714285714</td>
</tr>
<tr>
<td>mCherry 3 (B)</td>
<td>27.2</td>
<td>1.65</td>
<td>1.35</td>
<td>31.0857142857</td>
</tr>
<tr>
<td>mCherry 1 ©</td>
<td>25.3</td>
<td>1.89</td>
<td>2.46</td>
<td>28.9142857143</td>
</tr>
<tr>
<td>mCherry 2 ©</td>
<td>24.1</td>
<td>2.08</td>
<td>2.18</td>
<td>27.5428571429</td>
</tr>
<tr>
<td>mCherry 3 ©</td>
<td>24.0</td>
<td>2.05</td>
<td>2.54</td>
<td>27.4285714286</td>
</tr>
<tr>
<td>Reporter 1 (A)</td>
<td>18.1</td>
<td>1.57</td>
<td>1.42</td>
<td>20.6857142857</td>
</tr>
<tr>
<td>Reporter 2 (A)</td>
<td>18.8</td>
<td>1.55</td>
<td>1.36</td>
<td>21.4857142857</td>
</tr>
<tr>
<td>Reporter 3 (A)</td>
<td>21.1</td>
<td>1.55</td>
<td>1.33</td>
<td>24.1142857143</td>
</tr>
<tr>
<td>Reporter 1 (B)</td>
<td>21.6</td>
<td>2.05</td>
<td>2.31</td>
<td>24.6857142857</td>
</tr>
<tr>
<td>Reporter 2 (B)</td>
<td>20.7</td>
<td>2.09</td>
<td>2.17</td>
<td>23.6571428571</td>
</tr>
<tr>
<td>Reporter 3 (B)</td>
<td>21.8</td>
<td>2.06</td>
<td>2.63</td>
<td>24.9142857143</td>
</tr>
</tbody>
</table>
<ul class="bullets">
<li>Set up and re-ran Phusion PCR of pSB1C3 with same settings as 26/07</li>
<li>Streaked out 5 plates of pKDL071_mCherry (from colonies a, b, c) and pKDL071_Reporter (from colonies a and b) and put in 37°C incubator overnight</li>
</ul>
<h3><a id="Week_5_day_5_2807_710"></a>Week 5, day 5 (28/07)</h3>
<ul class="bullets">
<li>Checked the plates with pKDL017_mCherry and pKDL017_Reporter from 27/07</li>
<li>Ran the gel for 2nd PCR of PSB1C3 with new primers from 27/07</li>
</ul>
<ul class="bullets">
<li>Note: The exposure time might be too little (only 0.02), needs to be at least 0.35s.</li>
</ul>
<h3><a id="Week_6_day_1_3107_718"></a>Week 6, day 1 (31/07)</h3>
<ul class="bullets">
<li>Ran PCR for pSB1C3 + new UNS primers (Tm Calculator indicated annealing temperature was 67°C)
<ul class="bullets">
<li>6 samples at 62°C, 62.9°C, 64.9°C, 68°C, 70.5°C, and 72°C</li>
<li>One NTC at 68°C</li>
</ul>
</li>
<li>Ran gel of the results</li>
</ul>
<h3><a id="Week_6_day_3_0208_726"></a>Week 6, day 3 (02/08)</h3>
<ul class="bullets">
<li>Made electrocompetent MG 1655ΔLacI cells according to the electrocompetent cell protocol.</li>
<li>Made 80 glycerol stocks of the electrocompetent MG1655ΔLacI cells. Stored cells in -80°C freezer.</li>
</ul>
<h3><a id="Week_6_day_4_0308_731"></a>Week 6, day 4 (03/08)</h3>
<ul class="bullets">
<li>Vacufuged pKDL071 plasmids (w/ mCherry and Reporter) to concentrate DNA</li>
<li>Transformed electrocompetent MG1655ΔLacI cells (from 02/08) with RFP competence test kit and placed in 37°C incubator overnight</li>
<li>Resuspended vacufuged DNA samples in 7uL of nuclease free water to reach a concentration of 35ng/uL</li>
</ul>
<h3><a id="Week_6_day_5_0408_736"></a>Week 6, day 5 (04/08)</h3>
<ul class="bullets">
<li>Checked competence of MG1655ΔLacI cells (from 03/08) - the results were good.</li>
<li>Nanodropped mCherry and Reporter DNA samples from 03/08</li>
</ul>
<p>Table 1: Nanodrop results of vacufuged mCherry and Reporter DNA</p>
<table class="table table-striped table-bordered">
<thead>
<tr>
<th>Sample</th>
<th>ng/uL</th>
<th>260/280</th>
<th>260/230</th>
</tr>
</thead>
<tbody>
<tr>
<td>mCherry A1</td>
<td>67.3</td>
<td>2.74</td>
<td>1.00</td>
</tr>
<tr>
<td>mCherry A2</td>
<td>14.6</td>
<td>3.6</td>
<td>1.14</td>
</tr>
<tr>
<td>mCherry A3</td>
<td>17.2</td>
<td>5.23</td>
<td>0.83</td>
</tr>
<tr>
<td>mCherry B1</td>
<td>26.4</td>
<td>2.86</td>
<td>1.06</td>
</tr>
<tr>
<td>mCherry B2</td>
<td>38.6</td>
<td>2.76</td>
<td>1.16</td>
</tr>
<tr>
<td>mCherry B3</td>
<td>46.2</td>
<td>2.58</td>
<td>1.23</td>
</tr>
<tr>
<td>mCherry C1</td>
<td>56.2</td>
<td>2.61</td>
<td>1.23</td>
</tr>
<tr>
<td>mCherry C2</td>
<td>60.8</td>
<td>2.37</td>
<td>1.17</td>
</tr>
<tr>
<td>mCherry C3</td>
<td>48.4</td>
<td>2.57</td>
<td>1.23</td>
</tr>
<tr>
<td>Reporter A1</td>
<td>45</td>
<td>2.86</td>
<td>1.13</td>
</tr>
<tr>
<td>Reporter A2</td>
<td>47.5</td>
<td>2.69</td>
<td>1.06</td>
</tr>
<tr>
<td>Reporter A3</td>
<td>57.5</td>
<td>2.76</td>
<td>1.07</td>
</tr>
<tr>
<td>Reporter B1</td>
<td>81.9</td>
<td>2.69</td>
<td>1.1</td>
</tr>
<tr>
<td>Reporter B2</td>
<td>54.2</td>
<td>2.8</td>
<td>1.07</td>
</tr>
<tr>
<td>Reporter B3</td>
<td>54.3</td>
<td>2.63</td>
<td>1.11</td>
</tr>
</tbody>
</table>
<ul class="bullets">
<li>Performed Colony PCR of mCherry + Reporter construct</li>
<li>Ran gel of colony PCR results</li>
</ul>
<h3><a id="Week_7_day_2_0808_765"></a>Week 7, day 2 (08/08)</h3>
<ul class="bullets">
<li>Ran 12-well gel of 04/08 PCR products: pKDL071 plasmid in wells 2-8; pSB1C3 plasmid in wells 9-12
<ol>
<li>DNA ladder 10kb biobasic</li>
<li>Reporter A2</li>
<li>Reporter B3</li>
<li>mCherry A1</li>
<li>mCherry B2</li>
<li>mCherry C2</li>
<li>Positive control</li>
<li>Negative control</li>
<li>Sample 1</li>
<li>Sample 2</li>
<li>Sample 3</li>
<li>Negative control.</li>
</ol>
</li>
</ul>
<ul class="bullets">
<li>Repeated colony PCR of mCherry and reporter constructs in the pKDL071 plasmid
<ul class="bullets">
<li>PCR was run with samples mCherry A1, B3, and C1, and Reporter A1 and B3.</li>
<li>Samples were first diluted to concentrations of 1 ng/uL based on nanodrop results from 04/08.</li>
</ul>
</li>
<li>Gel electrophoresis of colony PCR of mCherry and reporter constructs.</li>
</ul>
<h3><a id="Week_7_day_3_0908_789"></a>Week 7, day 3 (09/08)</h3>
<ul class="bullets">
<li>Q5 PCR with Temperature Gradient (62-72°C) of PKDL071 with mCherry sample A1
<ul class="bullets">
<li>Samples 1-5 at 72.0°C, 70.0°C, 68.3°C, 64.3°C, 62.0°C, and 66.0°C</li>
<li>NTC at 66.0°C</li>
</ul>
</li>
<li>NEB OneTaq Hot Start PCR with temperature gradient of pSB1C3 to add UNSs with new primers. Tm Calculator indicated annealing temperature of 54°C.
<ul class="bullets">
<li>Samples 1-6 at 61.0°C, 59.1°C, 57.3°C, 53.5°C, 51.9°C, 51.0°C, 57.3°C</li>
<li>NTC at 57.3°C</li>
</ul>
</li>
<li>Transformation of MG1655ΔLacI with Reporter construct and mCherry construct respectively, both in pKDL071 bakcbone.</li>
<li>Gel eletrophoresis of both PCRs</li>
</ul>

<h3><a id="Week_7_day_4_1008_803"></a>Week 7, day 4 (10/08)</h3>
<ul class="bullets">
<li>Examined transformed MG1655ΔLacI with pKDL071 plasmid with insert from 09/08 (NC, Rep A, B, mCherry A, B, C)</li>
</ul>
<ul class="bullets">
<li>NEB OneTaq Hot Start PCR with temperature gradient of Christian’s inserts (using colony PCR primers to test if they are working)
<ul class="bullets">
<li>Made Master Mix (8 reactions) (50uL)
<ul class="bullets">
<li>80uL GC Reaction Buffer</li>
<li>8uL of 10uM dNTP</li>
<li>8uL of 10uM FWD</li>
<li>8uL of 10uM REV</li>
<li>4uL of OneTaq Polymerase</li>
<li>12uL of 100% DMSO (to make final concentration of 3%)</li>
</ul>
</li>
<li>Made NTC with 15uL of MM and 35uL nuclease free water</li>
<li>Added DNA to MM:
<ul class="bullets">
<li>7uL of DNA Template</li>
<li>238 uL nuclease free water</li>
</ul>
</li>
<li>Added 50uL final mix to 6 tubes and ran the PCR</li>
</ul>
</li>
<li>Gel electrophoresis of PCR products showed colony PCR primers weren’t working</li>
</ul>
<h3><a id="Week_7_day_5_1108_825"></a>Week 7, day 5 (11/08)</h3>
<ul class="bullets">
<li>Designed new pKDL071 colony PCR primers
<ul class="bullets">
<li>BLASTED against pKDL071 w/ UNSs, also BLASTED against <em>E.coli</em> K-12</li>
<li>Highest off target E-value was 0.46 for both FWD and RVS.</li>
<li>When analyzed with IDT OligoAnalyzer Tool the highest Delta G: -6.75 kcal/mole (Base Pairs3) was found for hetero-dimerization between fwd and rvs primers.)</li>
</ul>
</li>
</ul>
<p>“08-11 primer design.png”</p>
<ul class="bullets">
<li>Notes on how to interpret IDT OligoAnalyzer Output:
<ul class="bullets">
<li>For self-dimer analysis, click on ‘Self-Dimer’ to bring up a new window with each possible self-dimer your oligo can form. For each diagram you will be able to see the calculated delta G value for this secondary structure. If you have a strong delta G (-9kcal/mol or more negative) this oligo could be problematic.</li>
<li>Heterodimer analysis and hairpin formation is same as above</li>
</ul>
</li>
</ul>
<p>“08-11 analysis.png”</p>
<h3><a id="Week_8_day_4_1708_838"></a>Week 8, day 4 (17/08)</h3>
<ul class="bullets">
<li>Received new pSB1C3 and pKDL071 w/ UNS2/UNS3 FWD/REV primers</li>
<li>Phusion PCR with temperature gradient (62°C to 72°C) to put UNS primers onto pKDL071 and pSB1C3 backbone
<ul class="bullets">
<li>6 pKDL071 samples + NTC and 6 pSB1C3 samples + NTC</li>
</ul>
</li>
<li>Gel Electrophoresis of PCR products to determine size</li>
</ul>
<ul class="bullets">
<li>Redid Phusion PCR for pSB1C3 with new primers with increased template; also tested 2016 pSB1C3 primers with increased template
<ul class="bullets">
<li>Made Master Mix (MM1): (these were done for BOTH old (2016) and new (2.0) primers) for 8 reactions (50 μL each)
<ul class="bullets">
<li>80uL of 5x Phusion HF Buffer</li>
<li>8uL of 10mM dNTPs</li>
<li>20uL FWD primer</li>
<li>20uL REV primer</li>
<li>12uL of 100% DMSO</li>
<li>4uL Phusion polymerase</li>
</ul>
</li>
<li>Made NTC with 18uL of MM1 and 32uL of nuclease free water</li>
<li>Added DNA to MM1 to make MM2
<ul class="bullets">
<li>50uL (1ng/uL) template</li>
<li>174uL nuclease free H2O</li>
</ul>
</li>
<li>Made 6 50uL PCR tubes from MM2</li>
</ul>
</li>
<li>Gel electrophoresis of PCR products</li>
</ul>
<ul class="bullets">
<li>Dpn1 digest of pKDL071 to get rid of template</li>
<li>PCR purification and Nanodrop of digested pKDL071</li>
</ul>
<p>Table 1: Nanodrop results of purified Dpn1 digested pKDL071</p>
<p>| Sample    | ng/uL | 260/280   | 260/230 |<br>
| — | — | — |<br>
| 4 | 12.2  | 2.55  | 0.91 |<br>
| 6 | 11.6  | 2.80  | 0.75 |<br>
*Note: samples 3/4 and 5/6 were combined into sample 4 and 6, receptively.</p>
<ul class="bullets">
<li>Performed Gibson HiFi DNA assembly, adding 0.1 pmol of fragments and 0.05pmol of inserts</li>
</ul>
<p>Table 2: Reaction Component Table for Gibson HiFi DNA Assembly</p>
<table class="table table-striped table-bordered">
<thead>
<tr>
<th>Component</th>
<th>Reporter</th>
<th>mCherry</th>
</tr>
</thead>
<tbody>
<tr>
<td>pKDL071 + UNS BB</td>
<td>5.23 uL</td>
<td>5.23 uL</td>
</tr>
<tr>
<td>Reporter_mCherry</td>
<td>10.81 uL</td>
<td>-</td>
</tr>
<tr>
<td>Reporter_YFP</td>
<td>9.7 uL</td>
<td>-</td>
</tr>
<tr>
<td>LacILOV_mCherry</td>
<td>-</td>
<td>10 uL</td>
</tr>
</tbody>
</table>
<h3><a id="Week_8_day_5_1808_885"></a>Week 8, day 5 (18/08)</h3>
<ul class="bullets">
<li>Gibson plates from 17/08 (as well as the positive control supplied by Gibson kit) showed no colonies</li>
<li>PCR of pSB1C3 UNS with 2.0 UNS primers using a linearized pSB1C3 backbone to control for supercoil interference. We also ran a temperature gradient: 50-71°C. Phusion HF polymerase was used as well as 3% DMSO.</li>
<li>Gel electrophoresis of PCR products:</li>
</ul>
<p>“08-18 pSB1C3 gel.png”</p>
<ul class="bullets">
<li>
<p>Result: Primer Dimers and what appears to be a slightly larger product (we hypothesize that these could be the palindromic ends of our UNS primers (backbone complement) annealing and amplifying.</p>
</li>
<li>
<p>Redid Gibson assembly based on new Nanodrop of samples</p>
</li>
</ul>
<p>Table 1: Nanodrop results of purified Dpn1 digested pKDL071</p>
<p>| Sample    | ng/uL | 260/280   | 260/230 |<br>
| — | — | — |<br>
| 4 | 16.8  | 1.63  | 1.23 |<br>
| 6 | 16.1  | 1.63  | 0.72 |</p>
<p>Table 2: Reaction Component Table for Gibson HiFi DNA Assembly</p>
<table class="table table-striped table-bordered">
<thead>
<tr>
<th>Component</th>
<th>Reporter</th>
<th>mCherry</th>
</tr>
</thead>
<tbody>
<tr>
<td>pKDL071 + UNS BB</td>
<td>4 uL</td>
<td>4 uL</td>
</tr>
<tr>
<td>Reporter_mCherry</td>
<td>10.87 uL</td>
<td>-</td>
</tr>
<tr>
<td>Reporter_YFP</td>
<td>9.7 uL</td>
<td>-</td>
</tr>
<tr>
<td>LacILOV_mCherry</td>
<td>-</td>
<td>10 uL</td>
</tr>
</tbody>
</table>
<h3><a id="Week_9_day_1_2108_911"></a>Week 9, day 1 (21/08)</h3>
<ul class="bullets">
<li>OneTaq PCR of Christian’s miniprepped pKDL071 plasmid with backbone specific PCR colony primers to test them out with temperature gradient from 45-55°C</li>
<li>Gel electrophoresis of PCR product:</li>
</ul>
<ul class="bullets">
<li>Gel electrophoresis for assembled pKDL071 Reporter ® and mCherry (M) plasmids from 25/07 (1), 17/08 (2), and 18/08 (3)</li>
</ul>
<ul class="bullets">
<li>Made 5mL overnight LB+KAN cultures of pKDL071 mCherry colonies 1, 2, and 3 in preparation for miniprep the next day</li>
</ul>
<h3><a id="Week_9_day_2_2208_923"></a>Week 9, day 2 (22/08)</h3>
<ul class="bullets">
<li>Miniprepped Gibson assembly colonies A through C</li>
<li>Performed EZ-10 spin column PCR products purification kit of all PCR reactions using 2.0 primers on pSB1C3 backbone (samples 9-14 from 17/08 and samples 1-11 from 17/08)
<ul class="bullets">
<li>Purpose: to find out if the backbone + UNS was actually amplified during PCR, and was not visible due to low yield.</li>
<li>The purified products would then be combined to obtain enough product to run a gel extraction.</li>
</ul>
</li>
<li>Nanodropped PCR purified pKDL071+UNS samples 4 and 6 from 17/08 and miniprepped plasmid with mCherry in pKDL071 (colonies A,B and C)</li>
</ul>
<p>Table 1: Nanodrop results of PCR purified pKDL071+UNS and mCherry+pKDL071 colonies</p>
<table class="table table-striped table-bordered">
<thead>
<tr>
<th>Sample</th>
<th>ng/uL</th>
<th>260/280</th>
<th>260/230</th>
</tr>
</thead>
<tbody>
<tr>
<td>4</td>
<td>1.85</td>
<td>1.02</td>
<td>40.7</td>
</tr>
<tr>
<td>6</td>
<td>2.11</td>
<td>0.91</td>
<td>25.4</td>
</tr>
<tr>
<td>A</td>
<td>1.85</td>
<td>1.46</td>
<td>68.6</td>
</tr>
<tr>
<td>B</td>
<td>1.76</td>
<td>1.17</td>
<td>74.9</td>
</tr>
<tr>
<td>C</td>
<td>1.90</td>
<td>1.45</td>
<td>77.2</td>
</tr>
</tbody>
</table>
<ul class="bullets">
<li>OneTaq PCR of purified plasmid: 6 samples and 1 NTC</li>
<li>Gel electrophoresis of PCR purified pSB1C3</li>
</ul>
<p>“08-22 PCR purified pSC1C”</p>
<ul class="bullets">
<li>Gel electrophoresis of LacILOV mCherry from colonies and colony PCR products</li>
</ul>
<p>“08-22 OneTaq HotStart pKDL071 mCherry with 2.0 Colony PCR primers”</p>
<ul class="bullets">
<li>Gibson assembly of the reporter constructs</li>
</ul>
<h3><a id="Week_9_day_3_2308_950"></a>Week 9, day 3 (23/08)</h3>
<ul class="bullets">
<li>Vacufuged assembly from 22/08 and took Nanodrop each time</li>
<li>Ran a gel of miniprepped reporter</li>
</ul>
<h3><a id="Week_9_day_4_2408_954"></a>Week 9, day 4 (24/08)</h3>
<ul class="bullets">
<li>Gibson Assembly of mCherry_LacILOV insert and YFP_Reporter insert
<ul class="bullets">
<li>Note: we put the tubes in the PCR at 50°C instead of using thermocycler</li>
</ul>
</li>
<li>Constructed the light array</li>
<li>Transformed DH5α cells with Gibson assembled mCherry_LacILOV_YFP_reporter insert</li>
</ul>
<h3><a id="Week_9_day_5_2508_960"></a>Week 9, day 5 (25/08)</h3>
<ul class="bullets">
<li>Checked plates from yesterday and got 1 colony</li>
<li>Colony PCR of reporter insert (resuspended colony in water)</li>
<li>RE Digestion (pstI)</li>
<li>Gel electrophoresis in order: 2-log NEB ladder, Christian’s insert, Reporter insert, NTC</li>
</ul>
<h3><a id="Week_10_day_1_2908_968"></a>Week 10, day 1 (29/08)</h3>
<ul class="bullets">
<li>Phusion PCR with colony PCR primers on our Gibson Assembly product</li>
</ul>
<ul class="bullets">
<li>PCR of LacILOV_mCherry insert WITH restriction enzyme primers</li>
</ul>
<h3><a id="Week_10_day_2_3008_977"></a>Week 10, day 2 (30/08)</h3>
<ul class="bullets">
<li>Ran the gel for PCR purified Reporter and LacILOV.</li>
<li>Gibson Assembly of mCherry and YFP in pKDL071</li>
<li>RE Digest of reporter and pSB1C3</li>
<li>NEB Transformation protocol (E2621) for HIFI Gibson
<ul class="bullets">
<li>3 transformations: reporter, transformation, and assembly</li>
</ul>
</li>
<li>T4 DNA ligation of pSB1C3+reporter</li>
<li>Nanodropped LacILOV mCherry gel</li>
</ul>
<h3><a id="Week_10_day_3_3108_986"></a>Week 10, day 3 (31/08)</h3>
<ul class="bullets">
<li>RE Digest (XbaI, PstI) of LacILOV &amp; mCherry from gel</li>
<li>Transformation of rep_pSB1C3 into DH10B</li>
<li>Gibson assembly of reporter + pJDK071 with 1:1 ratio insert:BB</li>
</ul>
<h3><a id="Week_10_day_4_0109_991"></a>Week 10, day 4 (01/09)</h3>
<ul class="bullets">
<li>Colony PCR :
<ul class="bullets">
<li>Transformed DH10B cells with ligated pSB1C3 &amp; Reporter construct (3:1, reporter: Backbone) using OneTaq Hot Start polymerase.</li>
<li>Transformed reporter + pKDL071 HiFi DNA assembly with 1:1:1 of Reporter_YFP: Reporter_mCherry: pKDL071 w UNS, using OneTaq Hotstart</li>
</ul>
</li>
<li>Streaked reporter_pSB1C3 and reporter_pKDL071 colonies onto new LB+CAM plates</li>
<li>Transformed ligated reporter + pSB1C3 (1:3, reporter: Backbone) and ligated LacILOV_mCherry+ pSB1C3(3:1, insert:Backbone) into DH10B cells</li>
</ul>
<h3><a id="Week_11_day_2_0509_998"></a>Week 11, day 2 (05/09)</h3>
<ul class="bullets">
<li>OneTaq Hot Start Colony PCR on colonies from 01/09 (pSB1C3_reporter &amp; pSB1C3_mCherry_LacILOV (transformed into DH10Beta))</li>
<li>Phusion PCR: pSB1C3 backbone amplification using 2016 primers</li>
<li>Overnight culture of LacILOV_mCherry insert in pSB1C3 plasmid in DH5beta</li>
</ul>
<h3><a id="Week_11_day_3_0609_1003"></a>Week 11, day 3 (06/09)</h3>
<ul class="bullets">
<li>Redid Colony PCR w/ LacILOV_mCherry &amp; Reporter constructs and Phusion PCR for amplification of pSB1C3 backbone (with 2016 primers) from 05/09</li>
<li>Miniprepped our overnight culture of DH10Beta with LacILOV construct in pSB1C3</li>
<li>Phusion PCR of Gibson Mix with new UNS 2 REV and UNS 3 FWD primers</li>
</ul>
<h3><a id="Week_11_day_4_0709_1008"></a>Week 11, day 4 (07/09)</h3>
<ul class="bullets">
<li>Nanodrop results of pooled PCR purified samples from 06/09</li>
<li>Redid Phusion PCR from 06/09</li>
<li>RE Digest for PCR with RE primers of pSB1C3</li>
<li>PCR purification of digested pSB1C3</li>
</ul>
<h3><a id="Week_11_day_5_0809_1014"></a>Week 11, day 5 (08/09)</h3>
<ul class="bullets">
<li>Phusion PCR of LacILOV_mCherry insert from plasmid (inserted into both BBs) + directly from IDT stock to test UNS primers</li>
<li>Ran gel for PCR purified reporter construct with RE sites from 29/08
<ul class="bullets">
<li>Gel extract of single band on gel, following protocol from NEB Gel extraction kit</li>
<li>Nanodropped results</li>
<li>Phusion PCR of gel extracted reporter</li>
</ul>
</li>
<li>PCR purified PCR product from phusion PCR of reporter construct with RE sites using UNS primers (07/09)
<ul class="bullets">
<li>Nanodropped product</li>
<li>Gel of purified product</li>
</ul>
</li>
<li>Gibson Assembly of pKDL071 + reporter with vector: insert (2:1 ratio) from 22/08
<ul class="bullets">
<li>Transformed Gibson products following NEB Gel extraction protocol</li>
</ul>
</li>
<li>Ligated dephosphorylated + digested pSB1C3 BB from 07/09 and gel extraction product from today</li>
<li>LacILOV_mCherry colony and completely resuspended in microcentrifuge tube with LB+Kan</li>
</ul>
<h3><a id="Week_11_day_6_0909_1028"></a>Week 11, day 6 (09/09)</h3>
<ul class="bullets">
<li>We got colonies! From HiFi assembly of reporter+pKDL071 in 2:1 vector:insert ratio from 08/09
<ul class="bullets">
<li>Colony PCR to test</li>
</ul>
</li>
<li>Phusion PCR with UNS primers for (A) Gibson mix HiFi DNA assembly from 08/09, and B) IDT LacILOV_mCherry insert</li>
<li>Phusion PCR with RE primers for Gibson mix (HiFi assembly) from 08/09 with temperature gradient</li>
</ul>
<h3><a id="Week_12_day_1_1109_1034"></a>Week 12, day 1 (11/09)</h3>
<ul class="bullets">
<li>Colony PCR of reporter construct assembled 08/09 using UNS primers to eliminate non-specific binding possibility of colony PCR primers</li>
<li>Crossover PCR of reporter subpart YFP &amp; reporter subpart mCherry to assemble reporter construct</li>
<li>Inoculated 5ml overnight cultures with all reporter colonies</li>
<li>Performed LacILOV_mCherry end point measurement and start new assay</li>
</ul>
<h3><a id="Week_12_day_2_1209_1040"></a>Week 12, day 2 (12/09)</h3>
<ul class="bullets">
<li>Temperature gradient PCR with UNS 3 FWD and UNS 2 REV primers with LacILOV_mCherry in pKDL071 and pSB1C3</li>
<li>LacILOV_mCherry pSB1C3 miniprep</li>
<li>Overlap PCR of reporter inserts</li>
</ul>
<h3><a id="Week_12_day_3_1309_1045"></a>Week 12, day 3 (13/09)</h3>
<ul class="bullets">
<li>Miniprepping reporter construct in pKDL071</li>
<li>New UNS 4 fwd and rev primers</li>
<li>PCR purified reporter constructs from PCR amplification</li>
</ul>
<h3><a id="Week_12_day_4_1409_1050"></a>Week 12, day 4 (14/09)</h3>
<ul class="bullets">
<li>Phusion PCR of miniprep’d “reporter” plasmid in pKDL071 using colony PCR primers</li>
</ul>
<h3><a id="Week_12_day_5_1509_1053"></a>Week 12, day 5 (15/09)</h3>
<ul class="bullets">
<li>RE Digest of minipreped “reporter” plasmids R1, R2, and RA (used 10ul of plasmid in each rxn) from gibson assemblies with PKDL071 back bone.</li>
<li>Ran gel of PCR purified reporter inserts
<ul class="bullets">
<li>DNA Gel Extraction and Nanodropped results</li>
</ul>
</li>
<li>Phusion PCR temp gradient of reporter mcherry with UNS primers (65-55C)</li>
</ul>
<h3><a id="Week_13_day_1_1809_1059"></a>Week 13, day 1 (18/09)</h3>
<ul class="bullets">
<li>Redid reporter mCherry temperature gradient using PCR purified mCherry sample from 14/09</li>
</ul>
<h3><a id="Week_13_day_2_1909_1062"></a>Week 13, day 2 (19/09)</h3>
<ul class="bullets">
<li>Phusion PCR of LacILOV_mCherry in pKDL071 using UNS primers to test if PCR reagents are good</li>
</ul>
<h3><a id="Week_13_day_3_2009_1065"></a>Week 13, day 3 (20/09)</h3>
<ul class="bullets">
<li>PCR amplification of YFP reporter &amp; mCherry reporter from PCR purified product and gel extracted product.
<ul class="bullets">
<li>Used 1:10 diluted PCR purified YFP and mCherry and Gel extracted YFP and mCherry will as template for PCR</li>
</ul>
</li>
<li>Nanodropped Reporter Parts</li>
<li>Amplification of reporter insert from ALL previous Gibson Assemblies using UNS 3 FWD and UNS 2 REV</li>
<li>Final PCR: PCR of LacILOV_mCherry purified (1rxn + NTC + buffer)</li>
</ul>
<h3><a id="Week_13_day_4_2109_1072"></a>Week 13, day 4 (21/09)</h3>
<ul class="bullets">
<li>RE (BglII) digest of LacILOV_mCherry in pKDL071</li>
</ul>
<h3><a id="Week_13_day_5_2209_1075"></a>Week 13, day 5 (22/09)</h3>
<p>Note: From PCR results done on 21/09, PCRing after PCR purification is not usually successful due to incomplete amplicons from the first PCR</p>
<ul class="bullets">
<li>RE Digest results:
<ul class="bullets">
<li>Bgl II digest of LacILOV mcherry yielded unexpected results and digestion will be repeated today for a longer incubation period (~2.5h)</li>
<li>DpnI digest of PCR puified pKDL071; was not digested beforePCR purification, PCR purify after digest.</li>
</ul>
</li>
<li>Bgl II digest of LacILOV mcherry PKDL071, LacILOVmcherry pSB1C3, and Christan’s pKDL071 (digested for 2.5h)</li>
</ul>
<h3><a id="Week_14_day_1_2509_1083"></a>Week 14, day 1 (25/09)</h3>
<ul class="bullets">
<li>Ran temperature gradient PCR with colony PCR primers of LacILOV mCherry with Phusion HF to test if OneTaq gave artifacts</li>
<li>Started overnight culture of LacILOV_mCherry for assays</li>
<li>Designed primers to add RE sites (xbai and psti) to LacILOV, reason: be able to clone LacILOV into pSB1C3 for parts submission</li>
</ul>
<h3><a id="Week_14_day_2_2609_1089"></a>Week 14, day 2 (26/09)</h3>
<ul class="bullets">
<li>PCR (25ul rxn, 6 rxns) using colony PCR primers to check if successfully cloned reporter insert into pKDL071</li>
<li>Phusion PCR to amplify this to insert reporter_mCherry</li>
<li>DpnI digest of pKDL071 amplified backbone with UNS 2 at 5’ end and UNS 4 at 3’ end
<ul class="bullets">
<li>combined PCR samples 1&amp;2, 3&amp;4</li>
</ul>
</li>
<li>PCR purification and Nanodrop of dpnI digested pKDL071</li>
</ul>
<h3><a id="Week_15_day_3_2709_1096"></a>Week 15, day 3 (27/09)</h3>
<ul class="bullets">
<li>Gibson Assembly of Reporter_mCherry and pKDL071 BB with UNS 2 + 4
<ul class="bullets">
<li>Reporter_mCherry will be used from IDT stock</li>
<li>pKDL071 BB with UNS 2 and 4 will be used from sample 1 prepared 26/09</li>
</ul>
</li>
<li>Phusion PCR using UNS 4 FWD and UNS 3 REV to amplify pKDL071</li>
<li>Transformed of DH5a cells with our mCherry reporter in pKDL071
<ul class="bullets">
<li>Spreaded LB+Kan plates and placed in incubator overnight @ 37oC</li>
</ul>
</li>
</ul>
<h3><a id="Week_15_day_4_2809_1104"></a>Week 15, day 4 (28/09)</h3>
<ul class="bullets">
<li>Got 28 colonies from 27/09!</li>
<li>Colony PCR with OneTaq and colony PCR primers</li>
<li>DpnI digest of pKDL071 BB using UNS4 and UNS3 primers</li>
<li>PCR purification of DpnI digested pKDL071</li>
<li>Nanodrop of dpnI digested + PCR purified pKDL071 BB</li>
</ul>
<h3><a id="Week_15_day_5_2909_1111"></a>Week 15, day 5 (29/09)</h3>
<ul class="bullets">
<li>Started an overnight culture of colonies 1,4,5,6,14,18,26,29 (streaked onto new plates) in LB+KAN</li>
<li>Colony PCR of colony 29 from 28/09</li>
</ul>
<h3><a id="Week_15_day_6_3009_1115"></a>Week 15, day 6 (30/09)</h3>
<ul class="bullets">
<li>Miniprepped reporter_mCherry_pKDL071 colonies 1,4,5.6.14.18.26 &amp;29</li>
<li>PCR amplified those plasmids with UNS4 FWD &amp; UNS2 REV primers</li>
</ul>
<h3><a id="Week_16_day_1_0210_1119"></a>Week 16, day 1 (02/10)</h3>
<ul class="bullets">
<li>Phusion PCR to add UNS 3 to reporter mCherry in pKDL071 - will make it easier to add YFP reporter (Gibson)</li>
</ul>
<h3><a id="Week_16_day_2_0310_1122"></a>Week 16, day 2 (03/10)</h3>
<ul class="bullets">
<li>Received LacILOV Fwd and rev primers adding cut sites XbaI and PstI and YFP reporter stock from IDT</li>
<li>Phusion PCR for LacILOV PCR with RE sites</li>
<li>PCR amplification of reporter_mCherry colonies with UNS 4 FWD and pKDL071 UNS 3 rev to linearized plasmid and add UNS 3 for Gibson assembly of reporter_YFP</li>
</ul>
<h3><a id="Week_16_day_3_0410_1127"></a>Week 16, day 3 (04/10)</h3>
<ul class="bullets">
<li>Prepared samples for sequencing:
<ul class="bullets">
<li>Reporter_mCherry in pKDL071</li>
<li>LacILOV_mCherry in pSB1C3</li>
</ul>
</li>
</ul>
<h3><a id="Week_16_day_4_0510_1132"></a>Week 16, day 4 (05/10)</h3>
<ul class="bullets">
<li>DpnI digest of LacILOV AND digestion of LacILOV using XbaI/PstI</li>
<li>PCR purified digested LacILOV_with_RE sites</li>
</ul>
<h3><a id="Week_16_day_5_0610_1136"></a>Week 16, day 5 (06/10)</h3>
<ul class="bullets">
<li>Transformed ligated LacILOV in pSB1C3 to DH5B cells</li>
<li>Gibson assembly of reporter YFP with reporter_mCherry in pKDL071 from colony 1</li>
<li>Transformed into NEB Competent cells DH5a, plated on LB+AMP and incubated overnight</li>
</ul>
<h3><a id="Week_16_day_6_0710_1141"></a>Week 16, day 6 (07/10)</h3>
<ul class="bullets">
<li>Colony PCR</li>
</ul>
<h3><a id="Week_17_day_2_1010_1144"></a>Week 17, day 2 (10/10)</h3>
<ul class="bullets">
<li>Received mCherry reporter construct from IDT, dCas9 on CamR/TetR controlled plasmid from AddGene</li>
<li>DH5a-pKDL071-reporter colonies 13,20,21,22,23 were restreaked on LB+Kan and overnight cultures of these colonies were also made</li>
<li>dCas9 containing cultures were streaked onto an LB+Cam plate and put in 37°C incubator</li>
</ul>
<h3><a id="Week_17_day_3_1110_1149"></a>Week 17, day 3 (11/10)</h3>
<ul class="bullets">
<li>Miniprepped pKDL071_reporter plasmids from colonies 13, 20, 21</li>
<li>Made overnight culture of pdCas9 cells to miniprep the next day</li>
<li>Phusion PCR of reporter colonies with UNS 3 FWD and UNS 2 REV primers (7rxns, 25ul each)</li>
<li>Prepared sequencing primers for sequencing reporter construct in pKDL071
<ul class="bullets">
<li>seq 1,2,3,4,5,6</li>
<li>To amplify pSB1C3 out of LacILOV mCherry
<ul class="bullets">
<li>primers for UNS3 REV and UNS 2 FWD</li>
<li>UNS 5 REV - sgRNA + antiCRISPR</li>
<li>add UNS 3 beginning and UNS 2 REV for antiCRISPR</li>
</ul>
</li>
</ul>
</li>
<li>Transform MG1655ΔLacI cells with reporter pKDL071 colony 21 and 22
<ul class="bullets">
<li>Reporter mCherry pKDL071 was also transformed into MG1655ΔLacI as a +ve control for mCherry expression in LacILOV assays</li>
<li>RFP_pSB1C3 (10pg/ul) was used as +ve transformation control</li>
</ul>
</li>
<li>Miniprepped pdCas9 from overnight culture (1.5ml into 2 tubes)</li>
</ul>
<h3><a id="Week_17_day_5_1310_1164"></a>Week 17, day 5 (13/10)</h3>
<ul class="bullets">
<li>Phusion PCR for amplifying pSB1C3 backbone out of LacILOV with UNS 2 and UNS 3</li>
</ul>
<h3><a id="Week_17_day_6_1410_1167"></a>Week 17, day 6 (14/10)</h3>
<ul class="bullets">
<li>DpnI digest of pSB1C3 with UNS 2 and 3 PCR product</li>
<li>PCR purification of DpnI digested pSB1C3 UNS2 and 3 PCR product</li>
</ul>
<h3><a id="Week_18_day_2_1710_1171"></a>Week 18, day 2 (17/10)</h3>
<ul class="bullets">
<li>Phusion PCR of reporter_mCherry in pKDL071 and of AcrIIA4 and LacILOV with new UNS primers</li>
<li>DpnI digest of LacILOV and pKDL071 PCR product</li>
<li>PCR purification and Nanodrop of LacILOV, reporter_mCherry_pKDL071 with UNS 5, AcrIIA4, and reporter construct</li>
<li>Calculated Gibson Assembly of AcrIIA4, LacILOV, and sgRNA</li>
</ul>
<h3><a id="Week_18_day_3_1810_1177"></a>Week 18, day 3 (18/10)</h3>
<ul class="bullets">
<li>Transformed into DH5a cells and incubated at 37oC overnight (3plates each):
<ul class="bullets">
<li>AcrIIA4 in pSB1C3</li>
<li>sgRNA_reporter mCherry in pKDL071</li>
<li>LacILOV in pSB1C3</li>
</ul>
</li>
</ul>
<h3><a id="Week_18_day_4_1910_1183"></a>Week 18, day 4 (19/10)</h3>
<ul class="bullets">
<li>Colony PCR of 18/10 plates
<ul class="bullets">
<li>AcrIIA4 using colony rev/fwd primers</li>
<li>LacILOV using colony rev/fwd primers</li>
<li>sgRNA_reporter mCherry</li>
</ul>
</li>
</ul>
<h3><a id="Week_18_day_5_2010_1189"></a>Week 18, day 5 (20/10)</h3>
<ul class="bullets">
<li>Measurement of LacILOV_mCherry after diluting and growing overnight:
<ul class="bullets">
<li>First measurement was 12hrs after dilution</li>
<li>Note: LacILOV_pSB1C3 colony 9 was incubated from colony PCR sample after 5hrs from preparation as colony labeling on the plate was not clear. LacILOV_pSB1C3 colony 9 overnight culture didn’t grow and was not miniprep’d</li>
</ul>
</li>
<li>Miniprepped overnight cultures from colonies obtained after assembly</li>
</ul>
<h3><a id="Week_18_day_7_2210_1195"></a>Week 18, day 7 (22/10)</h3>
<ul class="bullets">
<li>Phusion PCR of :
<ul class="bullets">
<li>Reporter_mCherry_sgRNA_pKDL071 with UNS 4 FWD and pKDL071 UNS 3 REV (3rxn + NTC + extra) to assemble full construct</li>
<li>Reporter_mCherry_sgRNA_pKDL071 with UNS 5 REV and pKDL071 UNS 6 FWD (3rxn + ntc + extra) to assemble AcrIIA4 for antiCRISPR assays</li>
<li>Reporter_pKDL071 colony 21 &amp; 22 with UNS 3 FWD and UNS 4 REV (2rxn)</li>
<li>Reporter_pKDL071_colony 21 and 22 with UNS 4 FWD and UNS 2 REV (2rxn)</li>
</ul>
</li>
<li>DpnI digest of sgRNA_reporter mCherry for full construct and sgRNA_reporter mCherry_pKDL071 for AcrIIA4 assembly
<ul class="bullets">
<li>PCR purification of S1 and AS1</li>
<li>Nanodropped results</li>
</ul>
</li>
<li>Redid PCR of:
<ul class="bullets">
<li>Reporter_pKDL071 (colony 21 &amp; 22) with UNS 3 FWD and UNS 4 REV</li>
<li>Reporter_pKDL071 colony 21 and 22 with UNS 4 FWD and UNS 2 REV. (reporter_mCherry_pKDL071 will be used as +ve control) (3rxns)</li>
</ul>
</li>
<li>Gibson assembly:
<ul class="bullets">
<li>AcrIIA4_YFP subparts into sgRNA reporter_mCherry_pKDL071 (assembly of full construct)</li>
<li>AcrIIA4 alone into sgRNA_reporter_mCherry_pKDL071 with UNS 5 and 6</li>
</ul>
</li>
<li>Transformed MG1655ΔLacI cells with:
<ul class="bullets">
<li>sgRNA_rep_mCherry_pKDL071 colony 1 and pdCas9 (1ul of each plasmid will be used)</li>
<li>BBa_K523013 pSB1C3 to be used as +ve control for YFP found in plate 1 well 8H</li>
</ul>
</li>
<li>Colony PCR of:
<ul class="bullets">
<li>Full construct (AcrIIA4_YFP_sgRNA_repmCherry)</li>
<li>sgRNA rep_mCherry in pKDL071 with UNS 5 and 6 and AcrIIA4</li>
</ul>
</li>
<li>Streaked MG1655ΔLacI pdCas9 + sgRNA_pKDL071 on a new plate to get single colonies
<ul class="bullets">
<li>Started overnight culture of MG1655ΔLacI pdCas9 + sgRNA_pKDL071 for assays</li>
<li>Started overnight culture of MG1655ΔLacI YFP to trouble shoot YFP measurements</li>
</ul>
</li>
</ul>
<h3><a id="Week_19_day_2_2410_1220"></a>Week 19, day 2 (24/10)</h3>
<ul class="bullets">
<li>Miniprep and Nanodrop colonies F3 and F7, S2 and S5</li>
<li>Phusion PCR to take:
<ul class="bullets">
<li>Reporter construct out of pKDL071 (reporter colony #21)</li>
<li>Full construct out of miniprep’d pKDL071 (to Gibson into pSB1C3 tomorrow)</li>
</ul>
</li>
<li>DpnI digest of reporter construct [50ul rxn]</li>
<li>Gibson of reporter construct with pSB1C3…UNS 2 and 3</li>
</ul>
<h3><a id="Week_19_day_3_2510_1228"></a>Week 19, day 3 (25/10)</h3>
<ul class="bullets">
<li>Phusion PCR of pSB1C3 out of LacILOV pSB1C3 to add UNS 5 and extend UNS 3 for Gibson of full construct in pSB1C3</li>
<li>PCR purification of full construct (F3 and F7)</li>
<li>Colony PCR of reporter in pSB1C3 (twice)</li>
<li>Redid Phusion PCR from 24/10
<ul class="bullets">
<li>Full construct PCR out of pKDL071 (2rxns) using UNS primers</li>
<li>Full construct PCR out of pKDL071 (F3+F7) using phusion and pKDL071 colony PCR primers</li>
</ul>
</li>
</ul>
<h3><a id="Week_19_day_4_2610_1236"></a>Week 19, day 4 (26/10)</h3>
<ul class="bullets">
<li>Sent samples for sequencing:
<ul class="bullets">
<li>LacILOV_pSB1C3 (L3 + L4)</li>
<li>AcrIIA4_pSB1C3 (A1+A3+A7)</li>
<li>sgRNA_repmCherry_pKDL071 (S1)</li>
<li>AcrIIA4_sgRNA_repmCherry (S2+S5)</li>
</ul>
</li>
<li>Colony PCR of reporter_pSB1C3 Colonies 7-20
<ul class="bullets">
<li>Colonies 10,11,15 to be streaked onto new plates and miniprep’d</li>
</ul>
</li>
</ul>
<h3><a id="Week_19_day_5_2710_1245"></a>Week 19, day 5 (27/10)</h3>
<ul class="bullets">
<li>Miniprep of reporter_pSB1C3 colonies 10,11,15</li>
<li>XbaI and PstI digest of :
<ul class="bullets">
<li>LacILOV_pSB1C3 (L3 and L4)</li>
<li>AcrIIA4, pSB1C3 (A1, A3 and A7)</li>
<li>Reporter_pSB1C3</li>
</ul>
</li>
<li>Colony PCR (10ul/rxn, 14 rxns) + 1 buffer</li>
</ul>
<h3><a id="Week_19_day_7_2910_1253"></a>Week 19, day 7 (29/10)</h3>
<ul class="bullets">
<li>Minipreped full construct from overnight culture</li>
<li>Transformed full construct plasmids + pdCas9 in MG1655 delta LacI</li>
<li>Started overnight cultures:
<ul class="bullets">
<li>reporter_mCherry_pKDL071 + pdCas9 MG1655ΔLacI in minimum media + 0.2% glucose</li>
<li>reporter_mCherry_sgRNA_pKDL071 + pdCas9 MG1655ΔLacI in minimum media + 0.2% glucose</li>
<li>reporter_mCherry_sgRNA_AcrIIA4_pKDL071 + pdCas9 MG1655ΔLacI in minimum media + 0.2% glucose</li>
</ul>
</li>
<li>Streaked LacILOV_mCherry_pKDL071 MG1655ΔLacI on 4 LB+Kan plates to test LOKI</li>
<li>Grew :
<ul class="bullets">
<li>MG1655ΔLacI + YFP +ve to measure YFP</li>
<li>MG165ΔLacI repmCherry</li>
<li>pdCas9 cells</li>
</ul>
</li>
</ul>
<h3><a id="Week_20_day_1_3010_1266"></a>Week 20, day 1 (30/10)</h3>
<ul class="bullets">
<li>Restreaked 9 colonies (labeled) of MG1655ΔLacI_sgRNA_Acr_reportermCherry in pKDL071 backbone onto LB + CAM + KAN plates</li>
</ul>
</div>

		</div>

		<!-- Sidebar -->
		<div class="block sidebar">
			<div id="sidebar-box">
				<h4>Contents</h4>
				<div class="sidebar-minibox">
					<ul></ul>
				</div>
				<h4>Related Pages</h4>
				<div class="sidebar-minibox">
					<ul>
						<li> <a href="http://2017.igem.org/Team:Toronto/Experiments">Experiments</a></li>
						<li> <a href="http://2017.igem.org/Team:Toronto/InterLab">Interlab</a></li>
						<li> <a href="http://2017.igem.org/Team:Toronto/Results">Results</a></li>
					</ul>
				</div>
			</div>
		</div>
		<!-- Sidebar END -->

	</div>
</div>


</html>
{{Toronto/footer}}
